Tests/Admin/AdminTest.php

```diff
@@ -22,7 +22,7 @@ class AdminTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $this->contextManager = $this->getMock('Sonata\ClassificationBundle\Model\ContextManagerInterface');
+        $this->contextManager = $this->getMockForAbstractClass('Sonata\ClassificationBundle\Model\ContextManagerInterface');
```

Comment:
Would appreciate if you can create a new PR, so we have the same handling across all bundles.